src/en/models-cmr-scene-1.md

```diff
@@ -11,12 +15,12 @@ controller; used by the admin user, and consumed by multiple models.
 
```

Comment:
Maybe explain the cmr aspect of this in some way first. it is a bit confusing to go straight from this statement to creating a controller...

Reply:
OK, but it *is* an example scenario, and I *do* refer to the main page for background information.

Reply:
After re-reading it looks OK. The very first sentence says the scenario pertains to CMR (link). Do you really want me to re-defined what CMR is?

Reply:
No, I just think this doesn't start well - it mentions mediawiki briefly and then suddenly you are building a controller (or actually, bizarrely, adding credentials).
I think the whole thing would work a lot better if we actually set out what we were going to do, and lay out some requirements (e.g. In this example we have a controller called xxx and a model called yyy. Nobody cares that it is on aws. Then you can start the actual tutorial with something that matters, like adding mysql.

Reply:
OK, I thought your comment was for scenario #2. It makes more sense now.